docs/users-grants.rst

```diff
@@ -0,0 +1,169 @@
+
+Modifying a User
+----------------
+
+A ``User`` can be modified by changing one or more of its fields and then calling its ``save()`` method.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> user = api.select(User, 2345672)
+    >>> print(f"{user.first_name} {user.last_name}")
+    Arianne Martell
+    >>> print(user.phone)
+    800-555-0000
+    >>> user.phone = '888-555-9753'
+    >>> user.save()
+    <cbc_sdk.platform.users.User: id 2345672> @ https://defense.conferdeploy.net (*)
+    >>> print(user.phone)
+    888-555-9753
+
+**Note:** A user's *role* must be changed by modifying its *access grant,* detailed below.
+
+Creating a New User
+-------------------
+
+Creating a user may be done with the help of a *builder object,* which is returned from the ``User.create()``
+function.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> builder = User.create(api)
+    >>> builder.set_first_name('Samwell').set_last_name('Tarly')
+    <cbc_sdk.platform.users.User.UserBuilder object at 0x00000209B8123D00>
+    >>> builder.set_email('starly@example.com').set_phone('800-555-8008')
+    <cbc_sdk.platform.users.User.UserBuilder object at 0x00000209B8123D00>
+    >>> builder.set_role('psc:role::BETA_SYSTEM_ADMIN')
+    <cbc_sdk.platform.users.User.UserBuilder object at 0x00000209B8123D00>
+    >>> builder.build()
+
+Alternately, you may construct a *template object* (a Python ``dict``) that contains the user's information and
+create the user directly.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> user_template = {'first_name': 'Selyse', 'last_name': 'Florent', 'email': 'sflorent@example.com',
+    ...                  'phone': '877-555-9099', 'role_urn': 'psc:role::BETA_SYSTEM_ADMIN'}
+    >>> User.create(api, user_template)
+
+**Note:** A user that has just been created will *not* be visible in either the UI or in a ``User`` query as detailed
+above, until the intended user has responded to their invitation E-mail message, activated the account, and set
+a password.
+
+User Access Grants
+------------------
+
+Every user object has an *access grant* object associated with it, defining the access roles they are permitted to use.
+You can use the ``grant()`` method on a ``User`` to get the grant and inspect or modify it.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.platform import User
+    >>> user = api.select(User, 2345672)
+    >>> print(f"{user.first_name} {user.last_name}")
+    Arianne Martell
+    >>> grant = user.grant()
+    >>> print(grant.roles)
+    ['psc:role::BETA_SUPER_ADMIN']
+    >>> grant.roles = ['psc:role::BETA_SYSTEM_ADMIN']
```

Comment:
I would choose a different replacement role as I initially did not see the difference between the two admin roles. I would recommend an Analyst or View Only role

Reply:
I changed the roles used here to "system admin" and "view only."